ui/src/features/migration/RollingMigrationForm.tsx

```diff
@@ -971,7 +971,6 @@ export default function RollingMigrationFormDrawer({
                 .filter(vm => selectedVMs.includes(vm.id))
                 .map(vm => ({
                     vmName: vm.name,
-                    esxiName: vm.esxHost
```

Comment:
Why did we remove this?

Reply:
It is auto populated by backend anyways, so not sending it through UI is better option